### PR TITLE
19802- MVP Basic refund UI for ops only

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/19802

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2735-ke4k3nda.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2735-ke4k3nda.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2735-ke4k3nda.web.app

File: auth-web/src/views/auth/RefundView.vue [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+        </thead>
+        <tbody>
+          <tr
+            v-for="(item, index) in paymentLineItems"
+            :key="index"
+          >
+            <td>{{ index + 1 }}</td>
+            <td>{{ item.description }}</td>
+            <td>
+              <v-text-field
```
Comment (seeker25) [inline auth-web/src/views/auth/RefundView.vue:68]: Only the total is wired up right now I think

Comment (Jxio) [reply auth-web/src/views/auth/RefundView.vue:68]: image
it calculate total for each, but send them separately by LineItemId on payload

**Review by seeker25 — APPROVED** [review]

File: auth-web/src/views/auth/RefundView.vue
Comment (ochiu) [inline auth-web/src/views/auth/RefundView.vue:1]: does it make sense to be in views/auth? or should it be in pay?

**Review by ochiu — APPROVED** [review]

File: auth-web/src/views/auth/RefundView.vue
Comment (seeker25) [reply auth-web/src/views/auth/RefundView.vue:1]: I think `/pay/refund` will chunk it over

**Pull request merged**